engine/src/main/java/org/terasology/config/TelemetryConfig.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.config;
+
+/**
+ *  Configuration (authorisation) for telemetry system.
+ */
+public class TelemetryConfig {
+
+    private boolean enableTelemetry;
+
+    public boolean isEnableTelemetry() {
+        return enableTelemetry;
```

Comment:
I would rename the field to `telemetryEnabled`, `isTelemetryEnabled()` reads much better :)